backend/print_openapi_schema.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+import json
+
+from main import app
```

Comment:
Is it common to place such a file in the project? should we maybe have only one entry point and specify 'print-api-schema' as command-argument? Not sure what other projects normally do.

Reply:
I'll add a command. It's cleaner to not have a separate file.

Reply:
dda1f1f